src/mainboard/pcengines/apu2/mainboard.c

```diff
@@ -378,6 +378,19 @@ static void mainboard_enable(struct device *dev)
 
 	if (!check_sd3_mode())
 		sd_chip->sd_mode = 0;
+    
+	/* Enable IOMMU if activated in config file */
+	struct device* iommu_dev;
+	iommu_dev = pcidev_on_root(0, 2);
+
+	if (iommu_dev)
```

Comment:
```suggestion
	if (iommu_dev) {
```
In this project, opening brackets are placed in the same line as IF statement